angular-workspace/projects/ni/nimble-angular/rich-text-viewer/nimble-rich-text-viewer.directive.ts

```diff
@@ -0,0 +1,15 @@
+import { Directive } from '@angular/core';
+import type { RichTextViewer } from '@ni/nimble-components/dist/esm/rich-text-viewer';
+
+export type { RichTextViewer };
+
+/**
+ * Directive to provide Angular integration for the rich text viewer element.
+ */
+@Directive({
+    selector: 'nimble-rich-text-viewer'
+})
+
+export class NimbleRichTextViewerDirective {
+
```

Comment:
This class needs to be updated with `@Input` for `markdownValue`?

Reply:
Yes, it will be updated in the separate PR once https://github.com/ni/nimble/pull/1335 is complete. 

This implementation is done in [this branch](https://github.com/ni/nimble/blob/users/vivin/update-angular-directive-for-rich-text-viewer/angular-workspace/projects/ni/nimble-angular/rich-text-viewer/nimble-rich-text-viewer.directive.ts#L15) already.